pkg/store/instance.go

```diff
@@ -225,3 +232,64 @@ func AddGlobalFields(inst *Instance) (FormatData, error) {
 	}
 	return data, nil
 }
+
+// PrintInstances prints instances in a requested format to a given io.Writer.
+// Supported formats are "json", "yaml", "table", or a go template
+func PrintInstances(w io.Writer, instances []*Instance, format string) error {
+	switch {
```

Comment:
```suggestion
	switch format {
```

Reply:
Good point. Before, when I was doing regex, it needed to switch on cases; now, with just the format name, it can do a regular `switch format {...}`. Updated.